superset/views/sql_lab.py

```diff
@@ -188,12 +189,12 @@ def activate(  # pylint: disable=no-self-use
         owner_id = _get_owner_id(tab_state_id)
         if owner_id is None:
             return Response(status=404)
-        if owner_id != int(g.user.get_id()):
+        if owner_id != get_user_id():
             return Response(status=403)
 
         (
             db.session.query(TabState)
-            .filter_by(user_id=g.user.get_id())
+            .filter_by(user_id=get_user_id())
```

Comment:
Mixed types, though likely benign from the database's perspective—at least for MySQL.